docs/source/release_notes/release-notes.rst

```diff
@@ -84,21 +82,30 @@ develop
          - OkHttp clients (created with ``FeignOkHttpClients``) will no longer silently retry connections.
            We have already implemented retries, including retries from connection failures, at the Feign level in ``FailoverFeignTarget``.
            If you require silent retry, please contact the AtlasDB team.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/TODO>`__)
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2197>`__)
+
+    *    - |userbreak|
```

Comment:
Heh, I initially meant that this could be both improved and userbreak (rather than just userbreak). It is a good thing!